packages/react-dom/src/client/ReactDOM.js

```diff
@@ -739,14 +739,19 @@ const ReactDOM: Object = {
   unstable_flushControlled: DOMRenderer.flushControlled,
 
   __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
-    // For TapEventPlugin which is popular in open source
-    EventPluginHub,
```

Comment:
Technically this was used by TapEventPlugin. But it's already broken with 16.4 and has been deprecated: https://github.com/zilverline/react-tap-event-plugin#deprecated. So this doesn't matter in practice.